### DISCUSSION TITLE
Scrum Team Decision Log

### DISCUSSION THREAD

**Discussion by chowsanthony** [cdisc-org/ddf-core-poc#38] [Announcements]
2023-10-26: Stable model is v2.5 (https://github.com/cdisc-org/DDF-RA/releases/tag/v2.5), not v2.4.1

**Comment by rbakercdisc1** [comment]
Moved to version 2.6 and holding there. Stated at the 20-Nov-2023 meeting.

**Comment by rbakercdisc1** [comment]
For the assignment of the rules in the rule editor, the PBIs will have the Rule Editor number. The "Define" rule stage the number starting with 999xx is assigned for a temporary number. The information is placed in the[ Master Copy of USDMConformanceRulesNotes](url https://cdisc.sharepoint.com/:x:/r/sites/ddf/Shared%20Documents/DDF3a/Master%20Copy%20of%20USDMConformanceRulesNotes_ALL_2023-10-27.xlsx?d=w34732c00b4384d598cb238c881c36da9&csf=1&web=1&e=C2FPE0).

At the "Create" rule stage, the "Official" Rule Editor number is assigned.

**Comment by drewcdisc** [comment]
**From a retrospective on 2024-01-29:**

For classes that are located "under" StudyVersion in the model, rules will only check for conformance within a single version of the study (i.e. a single instance of a StudyVersion). 

This assumption aligns with the API design. 

It should also be noted that Dave IH has stated that only one version of a study should be included in a data set.
